security-proxy/src/main/java/org/georchestra/security/Proxy.java

```diff
@@ -205,8 +199,40 @@ public void init() throws Exception {
             for (String target : pTargets.stringPropertyNames()) {
                 targets.put(target, pTargets.getProperty(target));
             }
+
+            // Configure proxy permissions based on proxy-permissions.xml file in datadir
+            String datadirContext = georchestraConfiguration.getContextDataDir();
+            File datadirPermissionsFile = new File(String.format("%s%s%s", datadirContext,
+                    File.separator, "proxy-permissions.xml"));
+            if(datadirPermissionsFile.exists()){
+                logger.info("reading proxy permissions from " + datadirPermissionsFile.getAbsolutePath());
+                FileInputStream fis = null;
+                try {
+                    fis = new FileInputStream(datadirPermissionsFile);
+                    setProxyPermissions(Permissions.Create(fis));
+                    fis.close();
+                } catch(Exception ex){
+                      logger.error("Error during proxy permissions configuration from "
+                              + datadirPermissionsFile.getAbsolutePath());
+                } finally {
+                    fis.close();
+                }
+            }
+
             logger.info("Done.");
         }
+
+        // Proxy permissions not set by datadir
+        if (proxyPermissionsFile != null && proxyPermissions == null) {
+            Closer closer = Closer.create();
+            try {
+                final ClassLoader classLoader = Proxy.class.getClassLoader();
+                InputStream inStream = closer.register(classLoader.getResourceAsStream(proxyPermissionsFile));
```

Comment:
We should also be able to locate the permissions.xml file into the georchestra datadir

Reply:
my mistake, it is done just above ...